src/v2/Apps/Consign/OfferDetail/Components/StickyFooter.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+
+import { Flex, Link, Sans, Spacer } from "@artsy/palette"
+
+export const StickyFooter: React.FC = () => {
+  return (
+    <Flex
+      position="fixed"
+      bottom={0}
+      left={0}
+      height="46px"
+      width="100%"
+      borderTop="1px solid"
+      borderColor="black10"
+      alignItems="center"
+      justifyContent="center"
+      backgroundColor="white100"
+    >
+      <Sans size="2" color="black60">
+        Need help? <Link href="mailto:consign@artsy.net">Ask a question</Link>.
```

Comment:
I'm not sure why, but this link doesn't actually do anything. It looks right to me but maybe someone else sees something?